File: src/category_strings/q7_find_first_substring/FindFirstSubstring.py
from utils.LogUtil import LogUtil;

class FindFirstSubstring:
    # Write your code here 
    #
    # Determine the position of the first occurence of s in t.  
    # See README for details
    # @param s - any ASCII characters. Represents the search string
    # @param t - any ASCII characters. Represents a block of text
    # @return - position of the first occurence of s in t.
    def find_first_substring(self, s: str, t: str) -> int:
            return 10;

    def test(self):
        # Change these values to manually test your program
        str_input1 = "dog";
        str_input2 = "a cat and dog ran through the yard chasing a dog";
        accepted_response = 10;
        actual_response = self.find_first_substring(str_input1, str_input2);
        LogUtil().log_results_int("7: Find First Substring", accepted_response, actual_response);

File: src/category_strings/q7_find_first_substring/FindFirstSubstring.js
const LogUtil = require('../../utils/LogUtil.js');

/**
 * Determine the position of the first occurence of s in t.  
 * See README for details
 * @param {string} s - ny ASCII characters. Represents the search string
 * @param {string} t - any ASCII characters. Represents a block of text
 * @return {int} position of the first occurence of s in t.
 */
var findFirstSubstring = function(s, t) {
    // Write your code here
    return 10;
}

var test = function() {
    // Change these values to manually test your program
    let strInput1 = "dog";
    let strInput2 = "a cat and dog ran through the yard chasing a dog";
    let acceptedResponse = 10;
    let actualResponse = findFirstSubstring(strInput1, strInput2);
    new LogUtil().logResultsInt("7: Find First Substring", acceptedResponse, actualResponse);
}

module.exports = test;